src/transformStateWithClones.js

```diff
@@ -8,6 +8,29 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const result = [];
```

Comment:
```suggestion
  const statesHistory = [];
```
Name should describe variable, res/result/arr/array/el/elem/obj/object and etc are bad names